### PR TITLE
Fix numbering of old sessions

### LINKED ISSUE #1417: Many sessions have same SLUG
This should not be possible (on icchilisrv3)

screen shot 2018-10-02 at 04 44 54

### PR DESCRIPTION
**Pull request by lfaucon**
Closes #1417

### CODE REVIEW

File: frog/imports/api/sessions.js [hunk 1]
```diff
@@ -42,14 +42,9 @@ Meteor.methods({
       }).fetch();
       let nextNum = 1;
       if (prev && prev.length > 0) {
-        prev.sort((x, y) => y.slug - x.slug);
```
Comment (lfaucon) [inline frog/imports/api/sessions.js:45]: Here you sort alphabetically, that must have caused problems with "10" being less that "2"

**Comment by houshuang** [comment]
Nice, but I think this would make more sense (need to test it first):

```js
        const findIndex = s => parseInt(s.slug.split('-').pop(), 10);
        const prevNum = Math.max(...prev.map(findIndex));
        nextNum = prevNum + 1;
```

**Comment by houshuang** [comment]
@lfaucon Just made a small change, tested it, see if you approve before we merge.

**Comment by lfaucon** [comment]
Should be good

**Pull request merged**